src/aerospike_vector_search/client.py

```diff
@@ -325,56 +518,62 @@ def wait_for_index_completion(
         name: str,
         timeout: Optional[int] = sys.maxsize,
         wait_interval: Optional[int] = 12,
-        validation_checks: Optional[int] = 2,
+        validation_threshold: Optional[int] = 2,
     ) -> None:
         """
         Wait for the index to have no pending index update operations.
 
-        Args:
-            namespace (str): The namespace of the index.
-            name (str): The name of the index.
-            timeout (int, optional): The maximum time (in seconds) to wait for the index to complete.
+        :param namespace (str): The namespace of the index.
+        :type namespace: str
+
+        :param name (str): The name of the index.
+        :type name: str
+
+        :param timeout (int, optional): The maximum time (in seconds) to wait for the index to complete.
             Defaults to sys.maxsize.
-            wait_interval (int, optional): The time (in seconds) to wait between index completion status request to the server.
+        :type timeout: int
+
+        :param wait_interval: The time (in seconds) to wait between index completion status request to the server.
             Lowering this value increases the chance that the index completion status is incorrect, which can result in poor search accuracy.
+        :type wait_interval: int
 
         Raises:
             Exception: Raised when the timeout occurs while waiting for index completion.
-            grpc.RpcError: Raised if an error occurs during the RPC communication with the server while attempting to create the index.
+            AVSServerError: Raised if an error occurs during the RPC communication with the server while attempting to create the index.
```

Comment:
```suggestion
            AVSServerError: Raised if an error occurs during the RPC communication with the server while attempting to wait for the index.
```

I think there are more of these that I missed.

Reply:
Addressed mislabelled error messages in latest release.